docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='business-central-settings-creating-new-users-proc']
+= Creating new users
```

Comment:
I'd change this and the groups section to "Creating users" and "Creating groups". To create something entails it being new, so it's redundant. This is a common issue in most documentation, and in ours. The only exception, for me, is when you want to emphasize new vs existing.

Reply:
So true! Thanks for catching.